src/index.html

```diff
@@ -11,12 +11,71 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li class="nav__item">
+            <a
+              class="is-active"
+              href="#apple"
+            >
```

Comment:
Don't use spaces in `<a>` tag's `href` property. The `href` attribute in link on line 37 should not contain spaces around the anchor name.